tox.ini

```diff
@@ -37,7 +37,7 @@ commands = pylint caikit
 
 [testenv:imports]
 description = enforce internal import rules
-deps = pydeps
+deps = pydeps==1.12.3
```

Comment:
Version range?

Reply:
maaaybe, but this thing literally released half an hour ago and bricked the builds lol. This version works and has the functionality we need so I'd kinda rather just stick to it for stability for this tiny lil check